### PR TITLE
Add project issue templates and structure

### PR DESCRIPTION
**Pull request by leonardpahlke**
This PR adds a clear process to establish projects in the TAG. This is a more lightweight approach than we have with working groups. This should insure that, there is a clear path to start projects, clear project information (ownership, scope, deliverables etc.). The issue templates should help discussing a project proposal first and then helping the project leads along delivering the project.

cc @cncf/tag-env-sustainability 

PTAL @catblade @mkorbi

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/project-suggestion.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+### Additional info and comments
+
+- Reference to supporting material: `NONE`
+- Links to related site: `NONE`
+- Feel free to delete this section if you don't have more info: `NONE`
+
+## TODO
+
+Please complete these steps in order.
```
Comment (catblade) [inline .github/ISSUE_TEMPLATE/project-suggestion.md:47]: I'd remove the "in order".  Sometimes, did instance, I'm doing leads and proposal concurrently.

File: .github/ISSUE_TEMPLATE/project-tracking.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+#### Scope
+<!-- How much effort will this take? ok to provide a range of options if or "not yet determined"  -->
+
+...
+
+#### Goals and Non Goals
+<!-- How much effort will this take? ok to provide a range of options if or "not yet determined"  -->
+
+...
```
[the remark this answers is not in the document]
Comment (leonardpahlke) [reply .github/ISSUE_TEMPLATE/project-tracking.md:35]: hm yeah. perhaps this could also be part of the deliverables.

**Comment by mkorbi** [comment]
The deletions are cleanups?

**Comment by guidemetothemoon** [comment]
The changes LGTM for me, thanks @leonardpahlke ! I agree with Marlow's comments so I will just approve from my side since I believe you will follow up those comments anyway☺️

> The deletions are cleanups?

I think it's a cleanup. The plan is probably to add the information about the projects that we're already working on, like landscape and glossary, in subsequent PRs, once the structure is in place. @leonardpahlke please correct me if I'm wrong.

**Review by guidemetothemoon — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/project-suggestion.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+...
+
+### Impact
+<!-- Describe your hopes for how this would reduce risk for the cloud native ecosystem. Who will this help? How will it help them? -->
+
+...
+
+### Scope
+
```
Comment (mkorbi) [inline .github/ISSUE_TEMPLATE/project-suggestion.md:24]: We might add a description as well as an out-of-scope. Also to differentiate from Goals.

Comment (leonardpahlke) [reply .github/ISSUE_TEMPLATE/project-suggestion.md:24]: we got already a section to add a description. I will add a comment to mention out-of-scope things if that helps the project proposal to be more precise.

**Comment by leonardpahlke** [comment]
We need to add this to the governance. We dont need to be crazy detailed, just a high level overview. Mention our intentions to have clear ownership, expectations, deliverables so projects are setup for success. We can refer to the issue templates (project proposal / project tracking) with details. Would you like to take action item @caradelia?

**Pull request merged**